Src/Plugins/Security/SEC101.SecurePlaintextSecrets.json

```diff
@@ -46,6 +46,12 @@
           "ContentsRegex": "$SEC101/006.GitHubPat",
           "MessageArguments": { "secretKind": "GitHub personal access token (PAT)" }
         },
+        {
+          "Id": "SEC101/006",
+          "Name": "DoNotExposePlaintextSecrets/GitHubPat",
+          "ContentsRegex": "$SEC101/006.GitHubPat_2021_04_05",
```

Comment:
>GitHubPat_2021_04_05 [](start = 40, length = 20)

Call this GitHubPatCurrent

Reply:
Done.

---
In reply to: [608158353](https://github.com/microsoft/sarif-pattern-matcher/pull/366#discussion_r608158353) [](ancestors = 608158353)